Expand pkg README (#33)
PlasmaPy/hack-week: README.md

# 2022 Plasma Hack Week

[![License](https://img.shields.io/badge/License-BSD%202--Clause-blue.svg)](./LICENSE.md)
[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://docs.plasmapy.org/en/latest/CODE_OF_CONDUCT.html)
[![PyPI version](https://img.shields.io/pypi/pyversions/plasmapy?style=flat&logo=python)](https://img.shields.io/pypi/pyversions/plasmapy?style=plastic)

[![Documentation Status](https://readthedocs.org/projects/hack-week/badge/?version=latest)](https://docs.hack.plasmapy.org/en/latest/?badge=latest)

[![astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat)](http://www.astropy.org/)
[![astropy](http://img.shields.io/badge/powered%20by-PlasmaPy-ff2929.svg?style=flat)](http://www.plasmapy.org/)

> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/PlasmaPy/hack-week/HEAD)

Reviewer: Did we still want the binder link here?  If so, we'll need to update it.

Author: I honestly don't care either way.  I left it there because I retained the "Using Binder" section, and I retained the binder stuff because the `plasmapy` repo had it.

Author: If we remove the binder link, then we should also remove the "Using Binder" section.  I'm just going to leave it be, unless you specifically request it being removed.

Reviewer: I ended up getting this error when trying the binder link:
```
ERROR: Package 'hack' requires a different Python: 3.7.12 not in '>=3.8'
Removing intermediate container 0fb81a6b4ae4
The command '/bin/sh -c ${KERNEL_PYTHON_PREFIX}/bin/pip install --no-cache-dir .' returned a non-zero code: 1Built image, launching...
```
I'll create a separate issue for this, since it's probably not a bad idea to keep it available.

Author: I believe #34 will solve the issue.

Author: Nevermind, we now get this...

<img width="1581" alt="image" src="https://user-images.githubusercontent.com/29869348/178122709-c13adc53-139e-4f90-979d-70ca0d075995.png">

Author: See #35 for a solution.

> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/PlasmaPy/hack-week/HEAD)

Reviewer: ```suggestion
```

[2022-website]: https://hack.plasmpy.org/2022/about
[2022-schedule]: https://hack.plasmpy.org/2022/schedule
[2022-registration]: https://hack.plasmpy.org/2022/about
[2022-zoom]: https://us06web.zoom.us/meeting/register/tZMrd-6tqD0pEt21wrVM9DfQK91S1FgqqO_D
[Discord]: https://discord.gg/HdsZkp9M35
[resources]: ./resources

> This repository is used to house [resources] for th 2022 Plasma Hack Week

Reviewer: ```suggestion
This repository is used to house [resources] for the 2022 Plasma Hack Week
```

[here][2022-registration].

## Participating in the Event

The event will be hosted with [Zoom](https://zoom.us/).  Starting
Monday, July 11th, each day of the event will begin at 11:00 am ET
(8:00 am PT, 15:00 UTC).  The following link allows you to register for
the zoom meeting, [Hack Week Zoom][2022-zoom].  After filling out the
registration you will be emailed a link that gives access to every day
of the Hack Week.  This link will be unique to you, so please don't
share it with anyone.  However, feel free to share the invitation link above.

If you have any questions, then please feel free to email us at
hack@plasmapy.org or post to our [Discord] channel.

## Resources

Event resources can be found in the [resources] subdirectory.  This directory
will contain materials ranging from getting setup background materials
to event tutorials.  Please refer to the resources
[README](./resources/README.md) for more details.

## The `hack` Package

The primary goal of the 2022 Plasma Hack Week is to **train our scientists
so open-source software can thrive throughout the Plasma Community**.  To
do this, the Hack Week and participants will focus on developing a `hack`

> package.  The center of the package will focus on developing a plasma

Reviewer: ```suggestion
package.  The central focus of the package will be on developing a plasma
```

formulary, and additional plasma features will be driven by the

> participants interests.  While developing the `hack` package participants

Reviewer: ```suggestion
participants' interests.  While developing the `hack` package participants
```
